<launch>
    <node pkg="ros_gremsy" type="GimbalNode" name="ros_gremsy" output="screen">
        <rosparam command="load" file="$(find ros_gremsy)/config/config.yaml"/>
        <rosparam name="tf_prefix" value="gimbal_1"/>
    </node>
</launch>
